girder/settings.py

```diff
@@ -114,7 +114,7 @@ def _defaultRouteTable():
     @staticmethod
     @setting_utilities.default(SettingKey.SECURE_COOKIE)
     def _defaultSecureCookie():
-        return config.getConfig()['server']['mode'] == 'production'
+        return config.getServerMode() == PRODUCTION_MODE
```

Comment:
@AlmightyYakob #3063 removed this setting entirely. Can you merge master into this branch?